db/migrate/20220224191232_change_works_and_collections_auto_generate_thumbnail_to_thumbnail_selection.rb

```diff
@@ -0,0 +1,23 @@
+class ChangeWorksAndCollectionsAutoGenerateThumbnailToThumbnailSelection < ActiveRecord::Migration[6.1]
+  def up
+    add_column :works, :thumbnail_selection, :string, default: ThumbnailSelections::DEFAULT_ICON
```

Comment:
It may actually be better to use the string values here instead of the `ThumbnailSelections` constants. The reason being that, if in the future the `ThumbnailSelections` class is refactored or moved, this migration would no longer be executable. Even though it's way less DRY, it would be more future-proof to use `"default_icon"` instead of `ThumbnailSelection::DEFAULT_ICON`